eng/NoTargetsSdk.BeforeTargets.targets

```diff
@@ -1,9 +1,8 @@
 <Project>
 
   <PropertyGroup>
-    <!-- NoTargets SDK needs a TFM set. Use the latest .NETCoreApp TFM that is supported by the SDK.
-         Only set a default if the project isn't multi-targeting. -->
-    <TargetFramework Condition="'$(TargetFramework)' == '' and '$(TargetFrameworks)' == ''">$(NetCoreAppToolCurrent)</TargetFramework>
+    <!-- NoTargets SDK needs a TFM set. Set a default if the project doesn't multi target. -->
+    <TargetFramework Condition="'$(TargetFramework)' == '' and '$(TargetFrameworks)' == ''">$(NetCoreAppCurrent)</TargetFramework>
```

Comment:
Didn't we use `NetCoreAppToolCurrent` to make the transition to the new tfm (after the release) smoother or was just a mistake?